### DISCUSSION TITLE
Why is wasmer-Java so ignored?

### DISCUSSION THREAD

**Discussion by krisbitney** [wasmerio/wasmer#3672] [General]
A community contributor put up a PR more than a year ago that would add a basic feature--imports--to the project. It hasn't been reviewed.

I think it would be amazing to see some minimal attention on `wasmer-java`

**Comment by HeribertoCristofaro7vB** [comment]
It's great that you're interested in contributing to the wasmer-java project! Code review is an important part of the software development process, so it's understandable that you're concerned about the unreviewed PR.

One way to get the attention of the maintainers is to politely remind them about the PR and offer to help with the review process. You can also ask if there's anything specific they're looking for in terms of feedback or changes to the code.

Another option is to fork the project and implement the feature yourself. Once you've done so, you can submit your own PR and request a review. If the maintainers are still unresponsive, you may want to consider creating your own fork of the project and maintaining it independently.

Before you start contributing, it's a good idea to review the project's contribution guidelines and get familiar with the codebase. That way, you'll be better prepared to contribute high-quality code that aligns with the project's goals and standards.

Good luck with your contributions!
